### DISCUSSION TITLE
Support WakaTime

### DISCUSSION THREAD

**Discussion by ouuan** [cpeditor/cpeditor#671] [Ideas]
[WakaTime](https://wakatime.com/about) tracks time usage for programmers.

Reference: [WakaTime - Creating a Plugin](https://wakatime.com/help/creating-plugin)

The source of wakatime-cli is too big to be bundled:

![ncdu](https://user-images.githubusercontent.com/30581822/99055014-699b1e00-25d4-11eb-958a-bbc6b9b3e01a.png)

So we may need to let users install it manually, like CF Tool, etc.

**Comment by ShizuhaAki** [comment]
I think there is no need, because it is not exactly Competitive Programming related.

**Comment by ouuan** [comment]
> I think there is no need, because it is not exactly Competitive Programming related.

You can know how long you practice.

**Comment by ouuan** [comment]
This is moved back to #918.
